core/services/chainlink/config_test.go

```diff
@@ -315,6 +316,25 @@ func TestConfig_Marshal(t *testing.T) {
 			RPID:     ptr("test-rpid"),
 			RPOrigin: ptr("test-rp-origin"),
 		},
+		LDAP: toml.WebServerLDAP{
+			ServerTLS:                   ptr(true),
+			SessionTimeout:              models.MustNewDuration(15 * time.Minute),
+			QueryTimeout:                models.MustNewDuration(2 * time.Minute),
+			BaseUserAttr:                ptr("uid"),
+			BaseDN:                      ptr("dc=custom,dc=example,dc=com"),
+			UsersDN:                     ptr("ou=users"),
+			GroupsDN:                    ptr("ou=groups"),
+			ActiveAttribute:             ptr(""),
+			ActiveAttributeAllowedValue: ptr(""),
```

Comment:
Can we pass some example values?

Reply:
Good idea, updated one of the test cases now

Reply:
Can you update this case? This one is meant to be a full config, with every field set.